dmocks/dynamic.d

```diff
@@ -62,7 +62,15 @@ class DynamicT(T) : Dynamic
     ///
     override string toString()
     {
-        return _data.to!string();
+        alias DataType = typeof(_data);
```

Comment:
"data type" is ambiguous: the general term "data type" vs. "type of data".
"type of data" is no more concise than  `typeof(_data)`.
So, avoid the the `alias`?

    static if (is(typeof(_data) == class) && !hasFunctionAttributes!(typeof(_data).toString, "const"))
    {
        return (cast(Unqual!(typeof(_data))) _data).toString();
    }